content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -285,6 +285,7 @@ If third party tokens are not enabled, you should add the option `--set values.g
 
 Next, use the `istioctl x workload entry` command to generate:
 
+* `sidecar.env`: Contains metadata specific to each VM like istiod address to fetch CA.
```

Comment:
This is confusing since its not actually how its used. I think reality is more like "cluster.env: contains metadata developer A added. sidecar.env: contains metadata developer B added" :slightly_smiling_face: .

In the current VM installation method, it doesn't seem like there is a whole lot of motivation to split these files. It may be better suited to just consolidate all of them to cluster.env

Reply:
@stevenctl what's your idea? do we need to add the sidecar.env change to cluster.env?

Reply:
Agree with John. We can still support sidecar.env, but no real reason to generate both from our tool. 

https://github.com/istio/istio/pull/35340